src/App.tsx

```diff
@@ -1,14 +1,23 @@
-import { useState } from 'react'
+import React from 'react'
+import styled from 'styled-components'
+import FullScreenPage from '../src/components/FullScreenPage'
 
 
-function App() {
-  const [count, setCount] = useState(0)
+const App: React.FC=() =>{
+  
 
   return (
-    <>
-      
-    </>
+    <Wrapper>
+      <FullScreenPage children/>
```

Comment:
In React we use "children" props not in this way. 
Also, delete those components from here, we gonna use them when we work with pages.